app.js

```diff
@@ -73,11 +73,13 @@ document.querySelector("#book-form").addEventListener("submit", (e) => {
 
   const title = document.querySelector("#title").value;
   const author = document.querySelector("#author").value;
-  const book = new Book(title, author);
+  if (title !== '' && author !== ''){
+    const book = new Book(title, author);
 
-  Book.addBookToList(book);
-  Book.addBook(book);
-  Book.clearFields();
+    Book.addBookToList(book);
+    Book.addBook(book);
+    Book.clearFields();
```

Comment:
- [ ] when adding the same book twice it will remove both books when trying to delete only 1 of them. please fix this before requesting another code review.

Reply:
Please we did not use the same **filter()**, we use splice(). It just removes one book at the time

Reply:
Hi team,
the point is that when adding 2 books with the same title and author your application should remove just the one the user wants to remove instead of both